stdlib/public/core/StringCharacterView.swift

```diff
@@ -89,17 +91,28 @@ extension String {
       self._coreOffset = coreOffset
     }
   }
+  
+  /// A view of the string's contents as a collection of characters.
+  @_transparent // FIXME(sil-serialize-all)
+  public var _characters: _CharacterView {
+    get {
+      return CharacterView(self)
```

Comment:
This will still give a warning, right? Should be `_CharacterView`.

Reply:
Yep. Will fix in a separate PR for more warnings.